src/services/LeafValueCoder.ts

```diff
@@ -5,6 +5,8 @@ import { evenHex, prepend0x } from '../utils/helpers';
 import { FeedValue } from '../types/Feed';
 
 export class LeafValueCoder {
+  private static printableKeyRegexp = new RegExp(`^${FIXED_NUMBER_PREFIX}`);
```

Comment:
why regex? can we simple use split by `_`? or `.startWith()` to check if prefix is present?

Reply:
Because the come looks neater, it can be a different regex in the future. It's also very fast.